documentation/modules/con-requests-kafka-bridge.adoc

```diff
@@ -18,9 +18,9 @@ API request and response bodies are always encoded as JSON.
 Content-Type: application/vnd.kafka.v2+json
 ----
 
-* When performing producer operations, `POST` requests must provide `Content-Type` headers specifying the desired _embedded data format_, either `json` or `binary`, as shown in the following table.
+* When performing producer operations, `POST` requests must provide `Content-Type` headers specifying the desired _embedded data format_ of the messages produced. This can be either `json` or `binary`.
```

Comment:
```suggestion
* When performing producer operations, `POST` requests must provide `Content-Type` headers specifying the  _embedded data format_ of the messages produced. This can be either `json` or `binary`.
```